src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -212,7 +222,22 @@ public async Task<IActionResult> BulkEdit(WorkflowTypeIndexOptions options, IEnu
             return RedirectToAction(nameof(Index));
         }
 
-        public async Task<IActionResult> EditProperties(long? id, string returnUrl = null)
+        public async Task<IActionResult> Export(int id)
```

Comment:
Should be POST.

Reply:
Use `UnsafeUrl` (check for examples) instead of a manual form, perhaps that'll fix it already.

Reply:
Like this?
```razor
        <a asp-action="ResetSiteTheme" data-url-af="UnsafeUrl RemoveUrl" data-title="@T["Reset Site Theme"]" data-message="@T["Are you sure you want to reset the Site theme?"]" class="btn btn-danger">@T["Reset Site Theme"]</a>

```

Reply:
Wow, an amazing solution！
![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/e80c66b4-988b-4bc7-93d9-ef593915f15e)

Reply:
Awesome :).